### DISCUSSION TITLE
weird sudden location change to lat:0 lon:0

### DISCUSSION THREAD

**Discussion by xchetah** [stronnag/mwptools#146] [General]
Hi,
sometimes mwp shows that the drone was on location (0,0) , somewhere in the middle of the sea near Africa, I'm using multiwii 2.4 on arduino mega with u-blox 6m GPS receiver module.
does mwp check anything about the location data validity?
from what I understand, in order for this to happen ; multiwii should send location data with(FIX = true), in order for mwp to plot/show it , right?
do I need to modify multiwii gps code or is there a workaround using mwp?

**Comment by stronnag** [comment]
mwp uses 'MSP_RAW_GPS` and requires a 3D fix and >6 (or user defined) satellites. 

If the FC responds with 3D fix, >6satellites and "NULL Island" (0,0), the mwp believes it. mwp tries to avoid big jumps, but the logic may not work. 

As INAV does not behave in this fashion, and I haven't flown MW since 2015, without logs I cannot comment further.

**Reply by xchetah** [reply]
thanks, I'll modify the multiwii code then , to avoid this glitch

**Reply by xchetah** [reply]
as a side note, mwp does accept 4 sats with fix , I didn't change any settings related to this., I guess this contributes a little to this
